test/distributed/_composable/fsdp/test_fully_shard_init.py

```diff
@@ -453,5 +452,118 @@ def test_fully_shard_double_lazy_init(self):
             root_state._lazy_init()
 
 
+class TestFullyShardMetaDeviceInit(FSDPTestMultiThread):
+    """
+    Set ``torch.__future__.set_swap_module_params_on_conversion(True)`` using
```

Comment:
do you want to make it by default set to True in FSDP, and maybe after FSDPInstance destroy set it back to False, or you want user to manually set this flag?

Reply:
I think soon, we can have wrapper tensor subclasses always use `swap_tensors`, so setting this flag or not will not be a big deal. I think this makes sense since wrapper tensor subclasses would fail with `.data`, so `swap_tensors` is the only way.